pop-dark.toml

```diff
@@ -0,0 +1,160 @@
+# Pop Dark theme for the Helix Editor
+# Author: workingj<workingj@pm.me>
+# Repo: https://github.com/workingJ/helix-pop-theme
+# Version: 1.0
+# This theme is based on Nathaniel Webb's VSCodePopTheme <https://github.com/ArtisanByteCrafter/VSCodePopTheme>
+
+info = { fg = 'yellowH', bg = 'brownD' }
+hint = { fg = 'brownD', bg = 'yellowH', modifiers = ['bold'] }
+warning = { fg = 'brownD', bg = 'orangeW', modifiers = ['bold'] }
+error = { fg = 'brownD', bg = 'redE', modifiers = ['bold'] }
+'diagnostic.info'.underline = { color = 'yellowH', style = 'curl' } 
+'diagnostic.hint'.underline = { color = 'yellowH', style = 'curl' } 
+'diagnostic.warning'.underline = { color = 'orangeW', style = 'curl' } 
+'diagnostic.error'.underline = { color = 'redE', style = 'curl' } 
+'ui.background' = { bg = 'brownN' }
+'ui.window' = { bg = 'brownH', fg = 'brownD' } 
+'ui.gutter' = { bg = 'brownU' }
+'ui.text' = { fg = 'greyT' }
+'ui.text.focus' = { fg = 'orangeN' }
+'ui.text.info' = { fg = 'orangeH', bg = 'brownH' }
+'ui.cursor' = { fg = 'greyD', bg = 'orangeY' }
+'ui.cursor.insert' = { fg = 'orangeN', bg = 'orangeN' }
+'ui.cursor.select' = { fg = 'black', bg = 'orangeN' }
+'ui.cursor.match' = { fg = 'black', bg = 'blueD' }
+'ui.cursor.primary' = { fg = 'black', bg = 'orangeN' }
+'ui.selection' = { bg = 'blueH', fg = 'white' }
+'ui.selection.primary' = { bg = 'blueD', fg = 'white' }
+'ui.linenr' = { bg = 'brownU', fg = 'greyL' }
+'ui.linenr.selected' = { fg = 'orangeH' }
+'ui.cursorline' = { bg = 'brownH' }
+'ui.statusline.normal' = { fg = 'greyT', bg = 'brownD' }
```

Comment:
added these only lines, 
but did the commit through the web interface, so the whole files was replaced...